contributing/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.9.16-slim
+FROM python:3.9.16-slim-bullseye
```

Comment:
any particular advantage in using `bullseye` version?

Reply:
No reason. Only with `-slim` it was already `bullseye`, I just made it explicit as I'm using openjdk's image for `bullseye` as well.